src/main/java/io/jenkins/plugins/analysis/core/restapi/IssueApi.java

```diff
@@ -22,6 +28,19 @@
      */
     public IssueApi(final Issue issue) {
         this.issue = issue;
+        this.fileBlame = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5045840)